addons/product_margin/wizard/product_margin.py

```diff
@@ -22,6 +22,7 @@ def action_open_window(self):
         self.ensure_one()
         context = dict(self.env.context or {})
 
+        context.update({'create': False, 'edit': False})
```

Comment:
you can include them directly in the dict creation `dict(foo, bar=42)`

Reply:
Updated.